docs: Update WinUI 3 related article with the new discord link (#15715)
unoplatform/uno: doc/articles/uwp-vs-winui3.md

### Do you use 3rd-party libraries?

> While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us at our [Discord channel](https://platform.uno/discord).

Reviewer: Thanks, @agneszitte. I somehow missed this 🤔 Just for a bit of standardization, I suggest  
```suggestion
While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us on our [Discord Server](https://platform.uno/discord).
```

Author: thanks @morning4coffe-dev !
